benchmarks/nnls/solvers/nnls_scipy.py

```diff
@@ -0,0 +1,22 @@
+from benchopt.base import BaseSolver
+from benchopt.util import safe_import
+
+
+with safe_import() as solver_import:
+    from scipy import optimize
+
+
+class Solver(BaseSolver):
+    name = 'scipy'
+
+    install_cmd = 'pip'
+    requirements = ['scipy']
+
+    def set_objective(self, X, y):
+        self.X, self.y = X, y
+
+    def run(self, n_iter):
```

Comment:
maybe put a comment of why n_iter is not used but still kept in the function signature

Reply:
done, adding:
 
# n_iter is not used here:
# optimize.nnls(self.X, self.y, maxiter=n_iter) could produce
# a RuntimeError for some dataset like Boston.